sdcm/cluster.py

```diff
@@ -3255,6 +3255,9 @@ def stop_nemesis(self, timeout=10):
             nemesis_thread.join(timeout)
         self.nemesis_threads = []
 
+    def clear_termination_event(self):
```

Comment:
any reason not calling termination_event.clear() directly?

Reply:
Fixed